<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardiacStrain</class>
 <widget class="qMRMLWidget" name="CardiacStrain">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CardiacStrain.ui</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="11" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="advancedLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="myocardiiLabelLabel">
          <property name="text">
           <string>Myocardii Tissue Label</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="myoLabel">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="rvLabelLabel">
          <property name="text">
           <string>Right Ventricule Label</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="rvLabel">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="saveAHAButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save AHA</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="saveMotionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save Motion</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="saveStrainButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save Strain Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QWidget" name="inputOutputWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <layout class="QGridLayout" name="IOLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="loadSegButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Load Segmentation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="imgSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLSequenceNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="segSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLLabelMapVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="loadImgButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load CineMRI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Processing and Output</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPushButton" name="runStrainButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Measure Motion and Analyze Strain</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="statusLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Load images to start.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Visualization</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QPushButton" name="strainReportButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save Strain report</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QTabWidget" name="visualizationTabs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <property name="expanding" stdset="0">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="singlePlotTab">
      <attribute name="title">
       <string>Single Plot</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="singlePlotTypeLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="singlePlotTypeSelector">
         <item>
          <property name="text">
           <string>Strain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S. Rate</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="singlePlotDirectionLabel">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="singlePlotDirectionSelector">
         <item>
          <property name="text">
           <string>Radial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Long</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="singlePlotRegionLabel">
         <property name="text">
          <string>AHA Region</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="singlePlotRegionSelector">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Global</string>
         </property>
         <property name="maxCount">
          <number>2147483645</number>
         </property>
         <item>
          <property name="text">
           <string>Global</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Basal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Apical</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="triplePlotTab">
      <attribute name="title">
       <string>Triple Plot</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QComboBox" name="triplePlotRegionSelector2">
         <item>
          <property name="text">
           <string>Global</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Basal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Apical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="triplePlotRegionSelector3">
         <item>
          <property name="text">
           <string>Global</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Basal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Apical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="triplePlotDirectionSelector3">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Radial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Long</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="triplePlotDirectionSelector1">
         <item>
          <property name="text">
           <string>Radial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Long</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="triplePlotRegionSelector1">
         <item>
          <property name="text">
           <string>Global</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Basal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Apical</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="triplePlotTypeSelector2">
         <item>
          <property name="text">
           <string>Strain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S. Rate</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="triplePlotTypeSelector3">
         <item>
          <property name="text">
           <string>Strain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S. Rate</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="triplePlotDirectionSelector2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Radial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Long</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="triplePlotTypeSelector1">
         <item>
          <property name="text">
           <string>Strain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S. Rate</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="triplePlotRegionLabel">
         <property name="text">
          <string>AHA Region</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="triplePlotTypeLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="triplePlotDirectionLabel">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CardiacStrain</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CardiacStrain</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>imgSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
